net/shadowsocks-libev/files/shadowsocks-libev.init

```diff
@@ -307,7 +308,8 @@ validate_ss_server_section() {
 		'local_address:ipaddr' \
 		'local_ipv4_address:ip4addr' \
 		'local_ipv6_address:ip6addr' \
-		'bind_address:ipaddr'
+		'bind_address:ipaddr' \
+		'acl:string'
```

Comment:
Is the value supposed to be a path to the acl file?  Consider using something like `dst_ips_bypass_file:file`

Reply:
My guess is that the option should be set at the client side.  That is `validate_common_client_options_`, right?

Reply:
> Is the value supposed to be a path to the acl file? Consider using something like `dst_ips_bypass_file:file`

Changed type from `string` to `file`

> My guess is that the option should be set at the client side. That is `validate_common_client_options_`, right?

[According to docs](https://github.com/shadowsocks/shadowsocks-libev), ACL is only available in local and server mode. At least I use it for `ssserver`. So added to `validate_ss_server_section` and `validate_ss_local_section`.